### PR TITLE
EZP-25773: Cache warmup for urlAlias object

### PR DESCRIPTION
**Pull request by galileo**
Fixes: https://jira.ez.no/browse/EZP-25773

> Status: Ready for review

Added due to feature request in #1646

### CODE REVIEW

File: eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php [hunk 1]
```diff
@@ -206,6 +206,9 @@ public function lookup($url)
                 $this->logger->logCall(__METHOD__, array('url' => $url));
                 $urlAlias = $this->persistenceHandler->urlAliasHandler()->lookup($url);
                 $cache->set($urlAlias->id);
+
+                $urlAliasCache = $this->cache->getItem('urlAlias', $urlAlias->id);
```
Comment (joaoinacio) [inline eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php]: Hmm on a brief look I would expect the 'urlAlias', 'url' to (also) be set here, but could be wrong...

**Comment by andrerom** [comment]
+1

**Comment by bdunogier** [comment]
Must be rebased / adapted since #1646 has been merged.

**Comment by andrerom** [comment]
> Must be rebased / adapted since #1646 has been merged.

yes, but not a must for 1.3.1 as this is enhancement.

**Comment by galileo** [comment]
rebased @bdunogier @andrerom

**Pull request merged**